plugins/subnet/help.md

```diff
@@ -20,31 +24,91 @@ _This plugin does not contain a connection._
 
 ### Actions
 
+#### Check Address in Subnet
+
+This action is used to determine if the provided IP address is in the sub-net.
```

Comment:
```suggestion
This action is used to determine if the provided IP address is in the subnet.
```
Hyphen is unnecessary.

Reply:
I removed the hyphen, but now HelpValidator is failing. I checked this validator and it looks like the problem is with the plugin title itself. The validator recognizes the word "subnet" as a plugin title and expects it with a capital letter. I changed "subnet" to "Subnet" on that line and HelpValidator doesn't fail.